### PR TITLE
Fix api calls 

### PR DESCRIPTION
**Pull request by Jiay0928**
remove location service and set default location value

### CODE REVIEW

File: android/app/src/main/java/org/uwaterloo/subletr/utils/DefaultLocationData.kt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+package org.uwaterloo.subletr.utils
+
+val UWATERLOO_LONGITUDE = -80.5449f
+val UWATERLOO_LATITUDE = 43.4723f
```
Comment (leeminkyu16) [inline android/app/src/main/java/org/uwaterloo/subletr/utils/DefaultLocationData.kt]: Can you make this a const?

Comment (Jiay0928) [reply android/app/src/main/java/org/uwaterloo/subletr/utils/DefaultLocationData.kt]: Oh, true, okay

Comment (andr-dev) [inline android/app/src/main/java/org/uwaterloo/subletr/utils/DefaultLocationData.kt]: Yay constants

**Review by andr-dev — APPROVED** [review]
Nice!

**Review by abhaykshar — APPROVED** [review]
nice work!

**Pull request merged**